docs/names-and-ids.md

```diff
@@ -8,8 +8,8 @@ There are a number of names and IDs used in THIEF.  This table describes some of
 | - | - | - |
 | `servicePool` | string | Name of pool that a service app belongs to. |
 | `requestedServicePool` | string | Name of pool that a device app would like to pair with. |
-| `runId` | guid | Guid representing a running executable.|
-| `pairingId` | guid | Guid representing the pairing between a device app and a service app. |
+| `runId` | guid | Guid representing a single run of a device app.  Regenerated every time the device app launches. |
+| `serviceInstance` | guid | Guid representing a single run of a service app.  Regenerated every time the service app launches. |
```

Comment:
Suggest `serviceInstanceId`

Reply:
Good idea.  Wish I had thought of this sooner.  I'm going to do one more round of renaming and fit this in.